### PR TITLE
Url fixes

### PR DESCRIPTION
**Pull request by kenman345**
found out a lot of issues from travis, this tries to address most of them.

### CODE REVIEW

**Comment by kenman345** [comment]
might have an error, working on a fix

File: _data/cloud.yml [hunk 1]
```diff
@@ -10,8 +10,7 @@ websites:
       url: https://www.ctl.io/appfog/
       img: appfog.png
       bch: No
-      twitter: appfog
-      facebook: CenturyLinkCloud
+      twitter: centurylinkcld
```
Comment (t0p0lin0) [inline _data/cloud.yml]: I think there's a problem here. Should be 'appfog' for Twitter username, no?

File: _data/gambling.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ websites:
       othercrypto: Yes
 
     - name: Bitcoin Games
-      url: https://games.bitcoin.com
+      url: https://games.bitcoin.com/videopoker
```
Comment (t0p0lin0) [inline _data/gambling.yml]: Why change linking to the main landing page?

File: _data/cloud.yml (hunk 1 above)
Comment (kenman345) [reply _data/cloud.yml]: nope, I went through these, no twitter account should be listed.

File: _data/gambling.yml (hunk 2 above)
Comment (kenman345) [reply _data/gambling.yml]: because the first one redirects to the other, and that makes Travis annoyed so I just went direct

Comment (t0p0lin0) [reply _data/gambling.yml]: Makes sense!

File: _data/cloud.yml (hunk 1 above)
Comment (t0p0lin0) [reply _data/cloud.yml]: Fair enough, but there still is an incorrect Twitter handle (centurylinkcld) for appFog. I have the branch local and can push a quick change to remove the Twitter handle completely if you wish.

Comment (kenman345) [reply _data/cloud.yml]: its one and the same. I verified it myself because no one uses the handle centurylinkcloud anymore

Comment (t0p0lin0) [reply _data/cloud.yml]: Sorry, I'm stupid. It appears that 'centurylinkcld' is the parent company's Twitter handle. Still though, should this listing not use @appFog on Twitter?

Comment (kenman345) [reply _data/cloud.yml]: do they use that handle? I was tempted to remove it entirely as we have Centruylink in another section anyways and it doesnt look like this product is going anywhere

Comment (t0p0lin0) [reply _data/cloud.yml]: They do, I checked, and for consistency's sake, I suggest we use it for appFog. I'll push a quick commit through, then approve this PR.

**Pull request merged**